pkgs/development/libraries/highctidh/default.nix

```diff
@@ -0,0 +1,52 @@
+{
+  fetchgit,
+  fieldSize,
```

Comment:
Since this is a configuration parameter rather than a dependency, it would be somewhat idiomatic to give it a default value. This was never a firm rule (at least AFAIK), but it's already an observable pattern